libs/d3d12core/main.c

```diff
@@ -372,6 +372,8 @@ HRESULT STDMETHODCALLTYPE d3d12core_GetDebugInterface(d3d12core_interface *core,
 {
     TRACE("iid %s, debug %p.\n", debugstr_guid(iid), debug);
 
+    *debug = NULL;
```

Comment:
Needs `if (debug)` check I think. Most interfaces allow you to query for success without an out pointer.